Source/Core/DolphinQt/Config/MouseSettingsWidget.cpp

```diff
@@ -0,0 +1,142 @@
+          &MouseSettingsWidget::SaveSettings);
+  connect(m_snapping_distance_control, QOverload<double>::of(&QDoubleSpinBox::valueChanged), this,
+          &MouseSettingsWidget::SaveSettings);
+  connect(m_octagonal_mouse_jail_is_enabled_control, &QCheckBox::clicked, this,
+          &MouseSettingsWidget::SaveSettings);
+}
+
+void MouseSettingsWidget::LoadSettings()
+{
+  m_sensitivity_control->setValue(Config::Get(Config::MOUSE_SENSITIVITY));
+  m_snapping_distance_control->setValue(Config::Get(Config::MOUSE_SNAPPING_DISTANCE));
+  m_octagonal_mouse_jail_is_enabled_control->setChecked(
+      Config::Get(Config::MOUSE_OCTAGONAL_JAIL_ENABLED));
+}
+
+void MouseSettingsWidget::SaveSettings()
+{
+  Config::SetBaseOrCurrent(Config::MOUSE_SENSITIVITY, m_sensitivity_control->value());
+  Config::SetBaseOrCurrent(Config::MOUSE_SNAPPING_DISTANCE, m_snapping_distance_control->value());
+  Config::SetBaseOrCurrent(Config::MOUSE_OCTAGONAL_JAIL_ENABLED,
+                           m_octagonal_mouse_jail_is_enabled_control->isChecked());
+  Config::Save();
+  ciface::OctagonalMouseJail::GetInstance().RefreshConfigValues();
+}
+
+QHBoxLayout* MouseSettingsWidget::CreateSensitivityLayout()
+{
+  QHBoxLayout* sensitivity_layout = new QHBoxLayout{};
+
+  m_sensitivity_control = new QDoubleSpinBox{};
+  m_sensitivity_control->setRange(1.00, 9999.50);
+  m_sensitivity_control->setDecimals(2);
+  m_sensitivity_control->setSingleStep(0.5);
+  m_sensitivity_control->setWrapping(true);
+  QString sensitivity_tool_tip{
+      tr("Adjusts how quickly the mouse cursor moves the Cursor X/Y +/- inputs."
+         "\nChanges the distance at which your mouse snaps to notches. May make your snapping feel "
+         "weird."
+         "\n\n1.0(default) sensitivity maps the mouse to the full screen"
+         "\n\nRecommended value: 7.5")};
+  m_sensitivity_control->setToolTip(sensitivity_tool_tip);
+
+  QLabel* sensitivity_label = new QLabel{};
+  sensitivity_label->setToolTip(sensitivity_tool_tip);
+  sensitivity_label->setText(tr("Sensitivity"));
+
+  sensitivity_layout->addWidget(sensitivity_label);
+  sensitivity_layout->addWidget(m_sensitivity_control);
+
+  return sensitivity_layout;
+}
+
+QHBoxLayout* MouseSettingsWidget::CreateSnappingDistanceLayout()
+{
+  QHBoxLayout* snapping_distance_layout = new QHBoxLayout{};
+  m_snapping_distance_control = new QDoubleSpinBox{};
+
+  m_snapping_distance_control->setRange(0.00, 9999.50);
+  m_snapping_distance_control->setDecimals(2);
+  m_snapping_distance_control->setSingleStep(0.5);
+  m_snapping_distance_control->setWrapping(true);
+  QString snapping_distance_tool_tip{
+      tr("Distance (in pixels, can do partial pixels) around notches where cursor snaps to notch."
+         "\nMouse is only locked into notches and snapping will "
+         "only occur when the game is started and has focus."
+         "\n\n0.0(default) disables snapping")};
```

Comment:
This is a better phrasing IMO.
```
The distance (in pixels) where the cursor snaps to a notch of the virtual octagonal jail.
Setting it to "0.0" disable the snapping.
```
It's not necessary to specify that it accepts fractions of a pixel, since it is clear with the fact that the spinbox acceptthose values instead of just integers.